tests/aeroval/cams84.py

```diff
@@ -0,0 +1,164 @@
+
+# How to read auxiliary model variables (that cannot be read but need to be
+# computed). Entry under "fun" needs to be defined in ../eval_py/cube_read_methods.py
+MODEL_AUX_VARS = {
+    "vmro3": dict(vars_required=["mmro3"], fun="mmr_to_vmr"),
+    "vmrno2": dict(vars_required=["mmrno2"], fun="mmr_to_vmr"),
+}
+
+# Setup for models used in analysis
+MODELS = {
+    "IFS-OSUITE": dict(model_id="ECMWF_OSUITE", model_read_aux=MODEL_AUX_VARS),
+    "IFS-OSUITE-96h": dict(model_id="ECMWF_OSUITE_96H", model_read_aux=MODEL_AUX_VARS),
+    "IFS-CTRL": dict(model_id="ECMWF_CNTRL", model_read_aux=MODEL_AUX_VARS),
+    "IFS-CTRL-96h": dict(model_id="ECMWF_CNTRL_96H", model_read_aux=MODEL_AUX_VARS),
+}
+
+# Setup for available ground based observations (ungridded)
+
+VAR_OUTLIER_RANGES = {
+    "concpm10": [-1, 5000],  # ug m-3
+    "concpm25": [-1, 5000],  # ug m-3
+    "vmrno2": [-1, 5000],  # ppb
+    "vmro3": [-1, 5000],  # ppb
+}
+
+AERONET_SITE_FILTER = dict(station_name="DRAGON*", negate="station_name")
+OBS_GROUNDBASED = {
+    "AeronetL1.5-d": dict(
+        obs_id="AeronetSunV3Lev1.5.daily",
+        obs_vars=["ang4487aer", "od550aer"],
+        obs_vert_type="Column",
+        obs_filters=ChainMap(ALTITUDE_FILTER, AERONET_SITE_FILTER),
+        min_num_obs={"monthly": {"daily": 3}},
+    ),
+    "EEA-NRT-rural": dict(
+        obs_id="EEAAQeRep.NRT",
+        obs_data_dir=OBS_ACCESS.get("EEA_NRT_LOCAL"),
+        obs_vars=["concpm10", "concpm25", "vmro3", "vmrno2"],
+        obs_vert_type="Surface",
+        obs_outlier_ranges=VAR_OUTLIER_RANGES,
+        obs_filters=ChainMap(ALTITUDE_FILTER, EEA_NRT_RURAL_FILTER),
+    ),
+    "G-EEA-rural": dict(
+        obs_id="GHOST.EEA.daily",
+        obs_data_dir=OBS_ACCESS.get("GHOST_LOCAL_EEA_DAILY"),
+        obs_vars=["concpm10", "concpm25", "vmro3", "vmrno2"],
+        obs_vert_type="Surface",
+        obs_outlier_ranges=VAR_OUTLIER_RANGES,
+        obs_filters=ChainMap(ALTITUDE_FILTER, GHOST_BASE_FILTER, GHOST_RURAL_FILTER),
+    ),
+    "AirNow-rural": dict(
+        obs_id="AirNow",
+        obs_data_dir=None,
+        obs_vars=["concpm10", "concpm25", "vmro3", "vmrno2"],
+        obs_outlier_ranges=VAR_OUTLIER_RANGES,
+        obs_filters=ChainMap(ALTITUDE_FILTER, AIRNOW_RURAL_FILTER),
+        obs_vert_type="Surface",
+    ),
+    "AirNow": dict(
+        obs_id="AirNow",
+        obs_data_dir=OBS_ACCESS.get("AIRNOW_LOCAL"),
+        obs_vars=["concpm10", "concpm25", "vmro3", "vmrno2"],
+        obs_outlier_ranges=VAR_OUTLIER_RANGES,
+        obs_filters={**ALTITUDE_FILTER},
+        obs_vert_type="Surface",
+    ),
+    "MarcoPolo": dict(
+        obs_id="MarcoPolo",
+        obs_data_dir=OBS_ACCESS.get("MARCOPOLO_LOCAL"),
+        obs_vars=["concpm10", "concpm25", "vmro3", "vmrno2"],
+        obs_outlier_ranges=VAR_OUTLIER_RANGES,
+        obs_vert_type="Surface",
+    ),
+}
+
+
+# Setup for supported satellite evaluations
+OBS_SAT: dict = {}
+
+OBS_CFG = ChainMap(OBS_GROUNDBASED, OBS_SAT)
```

Comment:
Never used ChainMap before, is this dict-like? Otherwise the setter in `EvalSetup` might crash. I would probably use a dict directly, like suggested:
```suggestion
OBS_CFG = {**OBS_GROUNDBASED, **OBS_SAT}
```

Reply:
from the [python docs](https://docs.python.org/3/library/collections.html#collections.ChainMap)
> A ChainMap groups multiple dicts or other mappings together to create a single, updateable view. If no maps are specified, a single empty dictionary is provided so that a new chain always has at least one mapping.

And it passes all the tests, so it should work